docs/manifest-devhub.json

```diff
@@ -0,0 +1,1322 @@
+[
+	{
+		"title": "Block Editor Handbook",
+		"slug": "handbook",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/readme.md",
```

Comment:
Would be good to use relative links from the start to avoid the issue we had in the handbook.

Reply:
@youknowriad I'm not sure I understand here? The new DevHub has been ingesting absolute URLs without any issues. What was the problem that happened in the handbook?

Reply:
We want to stop targetting master at some point and have the docs use the stable version instead. If we use relative links, we should be able to use any branch from the consumer side.

Related #14303 #14739 and https://meta.trac.wordpress.org/ticket/4244

Reply:
Ahhhh, that makes perfect sense. @coffee2code any opinions/thoughts on this?

Reply:
I can bring the changes at https://github.com/WordPress/gutenberg/pull/14739 to this PR if trac/4244 is being worked on; otherwise, I'd both separately as to not block this one.

Reply:
I don't want to block the PR either, I just think this is something we should do for both the handbook and devhub.

Reply:
@coffee2code mentioned he'll work on the relative paths alongside this, so I've ported the changes in cc3ed5e

@youknowriad @chrisvanpatten I'm a bit confused about how do we plan to update the handbook going forward and wanted to flag two separate things that were asked to the meta team:

- Riad mentioned to use relative URLs. I understand this is to pinpoint the handbook to a specific branch (for example, latest WordPress release), and any updates to that branch will get reflected in the handbook (for example: in case we missed something in the docs, or want to expand them). Is that correct?

- Chris suggested to not sync the handbook on GitHub changes. I'm not sure how this would work or how we would trigger an update. Some clarity around this would be great.

I'm not familiar with the meta code that supports this, but wanted to flag the two things we're asking in case they conflict with each other.

Reply:
> (for example: in case we missed something in the docs, or want to expand them). Is that correct?

It's more related to the fact that the docs in master contain unreleased APIs and people shouldn't be reading these docs until WP releases a stable API (A version selector is better though)

> Chris suggested to not sync the handbook on GitHub changes.

I don't have opinions on the workflow itself when/how/where to sync docs. I just think it should be based on relative URLs and could target any branch from the consuming side.

Reply:
We don't have a plan specifically on how/when the updates will occur, but I do generally agree that being able to do a sync of a specific Git commit hash would be best, so we can specifically sync the documentation from the specific version of Gutenberg that is pulled into Core.

I expect—at least for the time being—this would be a manual process, and would likely mean opening a new Meta ticket with each core release providing the commit reference that will need to be synced in.

Reply:
(As long as we know that @coffee2code is handling building in relative link support in the short term, and we are comfortable shipping relative links on our side, then I see no reason not to make that change now, even if we don't have an exact process on how that sync will be triggered.)

Reply:
(That is to say, we know that _for now_ it will need to be Scott triggering a manual sync for us; ideally we can get a more scaleable process in place for the future but this should be good enough to get us through at least WP 5.3 while we strategize on a better approach.)

Reply:
That sounds good, thanks to both for clarifying. Added a note in the issue description linking the two core Trac tickets related to this one.

Reply:
Now that I've got better context and understand how relative paths will work [in practice](https://github.com/WordPress/gutenberg/pull/14739#issuecomment-491192067) and see the [concrete steps](https://github.com/WordPress/gutenberg/pull/15254#issuecomment-491211687), I've got a question: do we actually need to stop syncing content from GitHub? Why not to pinpoint the handbook contents to a specific branch? See [Slack conversation](https://wordpress.slack.com/archives/C02QB2JS7/p1557475613027000).

If we pinpointed the handbook to `wp/5.2` (just an example, needs to be discussed), the handbook will be effectively frozen. We would have a lightweight way to update the docs if needed, though, without having to update the manifest URL in DevHub. Less work for everyone: the branch used for generating the handbook would only change once per WordPress release.

Reply:
I would recommend pinning to a stable branch, e.g. `wp/stable` or `wp/devhub` rather than to a version-specific branch. That way we can reconfigure the Block Editor handbook to point to this new branch's manifest and leave it as is. Then it would just be a matter of someone on this end of things updating the branch whenever something needs changed.

The handbooks importer (for this and other handbooks) has cron-based auto-updates, which could be modified to support being disabled, but it'd just be easier all around to leave it in place (maybe with a longer interval).

BTW, the handbook syncing is automated. Any delays in seeing content changes reflected have stemmed from the need to make updates or fixes to importer to accommodate this new handbook.